pkg/runtime/field_export_reconciler.go

```diff
@@ -299,7 +301,7 @@ func (r *fieldExportReconciler) getSourcePathFromResource(
 ) (*string, error) {
 	obj, err := UnstructuredConverter.ToUnstructured(from.RuntimeObject())
 	if err != nil {
-		return nil, err
+		return nil, nil
```

Comment:
if the error is being ignored, logging before ignoring will be a better practice.

Reply:
Ah crap no this is a mistake I made when trying to change error handling elsewhere.